types/usercentrics-browser-ui/integration-interfaces.d.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * @file
+ * Everything from https://usercentrics.com/docs/web/implementation/ui/interfaces/
+ */
+
+import type { TCFUserDecisionOnVendor } from "./integration-types";
+
+/**
+ * A user's consent decision on all TCF options
+ * @see https://usercentrics.com/docs/web/implementation/ui/interfaces/#tcfdecisions
+ */
+export interface TCFDecisions {
+    /** A user's consent decision on TCF purposes */
+    purposes?: TCFUserDecisionOnPurpose[];
+
+    /** A user's consent decision on TCF special features */
+    specialFeatures?: TCFUserDecisionOnSpecialFeature[];
+
+    /** A user's consent decision on TCF vendors */
+    vendors?: TCFUserDecisionOnVendor[];
+}
+
+/** @see https://usercentrics.com/docs/web/implementation/ui/interfaces/#basetcfuserdecision */
+export interface BaseTCFUserDecision {
+    /** Indicates if the user gave consent (true) or denied consent (false) */
+    consent?: boolean;
+
+    /** The id of the purpose/vendor the consent decision belongs to */
+    id: number;
```

Comment:
Hey, shouldn't be the id of type string?

https://usercentrics.com/docs/web/features/api/control-functionality/#updatecategoriesconsents

Reply:
Hmm probably you're right, but https://usercentrics.com/docs/web/implementation/ui/interfaces/#basetcfuserdecision documents it as `number`. I think it would be fine to allow both. @coltmans would you like to send a PR for that?

Reply:
Hey @FloEdelmann I guess you are pinged anyway as owner. [PR](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71479) is ready. Thanks for reviewing :)